pontificia-universidade-catolica-do-parana-abnt.csl

```diff
@@ -1,216 +1,465 @@
+      <term name="no date" form="short">s. d.</term>
     </terms>
-    <style-options punctuation-in-quote="false"/>
   </locale>
   <macro name="container-contributors">
+    <!-- Macro responsavel por mostrar os nomes dos organizadores ou editores -->
     <choose>
-      <if type="chapter">
+      <if type="chapter entry-dictionary entry-encyclopedia">
         <names variable="container-author" delimiter=", ">
-          <name name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="always">
+          <name name-as-sort-order="all" sort-separator=", " delimiter="; " delimiter-precedes-last="always">
             <name-part name="family" text-case="uppercase"/>
-            <name-part name="given" text-case="capitalize-first"/>
+            <name-part name="given"/>
           </name>
-          <label form="short" prefix=" (" suffix=")"/>
+          <label form="short" prefix=" (" suffix=".)"/>
+          <et-al font-style="italic"/>
           <substitute>
             <names variable="editor"/>
+            <!-- ABNT 6023:2018 - quando for organizador -->
             <names variable="collection-editor"/>
+            <!-- ABNT 6023:2018 - quando for editor, mesmo que apenas do livro no todo -->
           </substitute>
         </names>
       </if>
     </choose>
   </macro>
-  <macro name="translator">
-    <group>
-      <names variable="translator" delimiter=", ">
-        <label suffix=": " text-case="capitalize-first"/>
-        <name delimiter=", " sort-separator=" " delimiter-precedes-last="always">
-          <name-part name="given"/>
-          <name-part name="family" text-case="capitalize-first"/>
-        </name>
-        <et-al font-style="normal"/>
-      </names>
-    </group>
+  <macro name="secondary-contributors">
+    <!-- Macro responsavel por mostrar os nomes dos organizadores do livro no todo quando um capítulo de livro -->
+    <choose>
+      <if type="chapter" match="none">
+        <names variable="editor" delimiter="; " prefix=" (" suffix=")">
+          <name and="symbol" delimiter=", "/>
+          <label form="short" prefix=", " text-case="capitalize-first" suffix="."/>
+        </names>
+      </if>
+    </choose>
   </macro>
   <macro name="author">
-    <names variable="author">
-      <name name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="always">
-        <name-part name="family" text-case="uppercase"/>
-        <name-part name="given" text-case="capitalize-first"/>
-      </name>
-      <label form="short" prefix=" (" suffix=".)" text-case="uppercase"/>
-      <substitute>
-        <names variable="editor"/>
-        <text macro="title"/>
-      </substitute>
+    <!-- Macro responsavel por mostrar o autor -->
+    <choose>
+      <if type="broadcast" match="any">
+        <!-- para transmissões é apresentador -->
+        <text value="Apresentado por "/>
```

Comment:
You've reintroduced the use of "value". Please replace any text value... with a term or label.
Same goes for hardcoded affixes.